### ISSUE TITLE
is this vulnerable to the Log4j zero-day?

### ISSUE THREAD

**Issue by floppydiskette** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#548]
as this is a java application, i am worried about potential security holes before this gets implemented into our company. i have recently heard of this new exploit on the news and would like more information before proceeding.

best,
floppydiskette, costco CTO

**Comment by ayangd** [comment]
As long as this project doesn't use the Log4j library, it should be fine.

**Comment by Tylersuard** [comment]
I just checked.  Fortunately, we have not updated our software since before Log4J was created.  We plan on skipping the affected package, and updating to Log5J when available.

**Comment by chebyshevvv** [comment]
Maybe log8j.I researched everywhere in internet but couldn’t find any log1j or log2j
or log3j.I guess that name of log4j come from log(1+3)j.So the next version name should be log(4+3)j.

**Comment by MrChip53** [comment]
> Maybe log8j.I researched everywhere in internet but couldn’t find any log1j or log2j or log3j.I guess that name of log4j come from log(1+3)j.So the next version name should be log(4+3)j.

That's a good observation for the naming patterns of the logXj library, but wouldn't it be log7j then?

**Comment by SleepyYui** [comment]
I would wait for log20j since there will probably be other exploits regarding the Logging tool. Until log20j there will be some flaws still, but in 16 generations errors arent existing anymore because noone will use Java then.

Sincerely, log4j exploiter
Cheers

**Comment by chebyshevvv** [comment]
> > Maybe log8j.I researched everywhere in internet but couldn’t find any log1j or log2j or log3j.I guess that name of log4j come from log(1+3)j.So the next version name should be log(4+3)j.
> 
> That's a good observation for the naming patterns of the logXj library, but wouldn't it be log7j then?

Yes , there is a flaw , but never mind , this is the fixed naming pattern version of sequence : log(0+4)j , log(0+4+4)j , log(0+4+4+4)j ......

**Comment by Kyykek** [comment]
I think it's quite obvious that log4j is referring to the logarithm with base 4 of j. Thus, in order to increment the number, the new name should be log4(4j). The parentheses are required as well for distinction between base and argument.
